engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/details/erratum-content-hosts.controller.js

```diff
@@ -69,7 +68,12 @@ angular.module('Bastion.errata').controller('ErratumContentHostsController',
             searchStatements = _.map(errataIds, function(errataId) {
                 return searchTerm + ' = "' + errataId + '"';
             });
-            return searchStatements.join(" or ");
+
+            hostSearchString = searchStatements.join(" or ");
+
+            IncrementalUpdate.setHostSearchString(hostSearchString);
```

Comment:
this looks like its ignoring the user search on that page.  (its also ignoring the environment filter, maybe we should get rid of it?)

Reply:
I believe that was also the case prior to my changes.